_docs/howto/configuring-forseti.md

```diff
@@ -46,6 +46,81 @@ If you change a configuration, such as by changing which pipeline or scanner to
 run, you'll need to restart the appropriate module so it picks up the new
 configuration change.
 
-## What's next
 
--   Read about [configuring Inventory]({% link _docs/quickstarts/inventory/#configuring-inventory %}).
+### Configuring Inventory
+
+{% include _howto/config_inventory.md %}
+
+#### Setting up batch mode
+
+You can run Forseti Inventory in batch mode to execute multiple inventory
+pipelines to in the same run. To specify which pipelines to run:
+
+1.  Open `forseti-security/configs/forseti_confs.yaml`.
+1.  Navigate to the `inventory` section.
+1.  Edit the `enabled` flag for the appropriate pipelines. Set the flag to
+    `true` to enable a pipeline, or `false` to disable the pipeline.
+
+````
+inventory:
+
+    pipelines:
+        - resource: appengine
+          enabled: true
+        - resource: backend_services
+          enabled: true
+        - resource: bigquery_datasets
+          enabled: true
+        - resource: buckets
+          enabled: true
+        - resource: buckets_acls
+          enabled: true
+        - resource: cloudsql
+          enabled: true
+        - resource: firewall_rules
+          enabled: true
+        - resource: folder_iam_policies
+          enabled: true
+        - resource: folders
+          enabled: true
+````
+
+When you're finished making changes, run the following command to update your
+`forseti_inventory` configuration:
+
+````
+forseti_inventory --config_path path/to/forseti_conf.yaml
+````
+
+### Configuring Scanner
+
+You can run Forseti Scanner in batch mode to execute multiple scanners in the
+same run. To specify which scanners to run in a batch:
+
+1.  Open `forseti-security/configs/forseti_confs.yaml`.
+1.  Navigate to the `scanner` section.
+1.  Edit the `enabled` flag for the appropriate scanners. Set the flag to `true`
+    to enable a scanner, or `false` to disable the scanner.
+
+````
+scanner:
```

Comment:
same comment as above about leaving this part out. As we add new scanners, it will have to be updated and we will likely forget.